util/src/main/java/io/grpc/util/TransmitStatusRuntimeExceptionInterceptor.java

```diff
@@ -219,6 +219,25 @@ public void run() {
       });
     }
 
+    /**
+     * A hint to the call that specifies how many bytes must be queued before
```

Comment:
Delete this Javadoc, it will be inherited. Basically remove it anywhere we are `@Override`ing and the javadoc remains the same as the parent class.